.github/workflows/daily-testgrid.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
 
     - name: testgrid-queue
       env:
-        TESTGRID_API: "https://api.testgrid.staging.kurl.sh/"
```

Comment:
we should run these 7 days a week ^^ the cron spec is 1-5 but images might change over the weekend